src/Session.php

```diff
@@ -141,9 +144,7 @@ public function getSelectorsHandler()
     public function visit($url)
     {
         // start session if needed
```

Comment:
You can remove `// start session if needed` comment from all 3 methods, because it's already present in `start` method.